backend/crud.py

```
@@ -0,0 +1,32 @@
+from typing import List, Sequence
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import

(F401)